cirq-google/cirq_google/engine/engine_processor.py

```diff
@@ -110,8 +110,11 @@ def get_sampler(
         Returns:
             A `cirq.Sampler` instance (specifically a `engine_sampler.ProcessorSampler`
             that will send circuits to the Quantum Computing Service
-            when sampled.1
+            when sampled.
         """
+        if processor := self._inner_processor():
```

Comment:
The result of `_inner_processor()` is `QuantumProcessor`, which should never be falsey (otherwise type should be `| None`).